app/javascript/app/components/nav/nav-component.jsx

```diff
@@ -4,6 +4,7 @@ import Link, { NavLink } from 'redux-first-router-link';
 import cx from 'classnames';
 import environmentalAffairsLogo from 'assets/environmental-affairs-logo.png';
 import SAflag from 'assets/south-africa-flag.png';
+import { ENVIRONMENTAL_AFFAIRS_URL } from '../../constants';
```

Comment:
Sorry for being fussy, but I prefer not to have relative imports.
Last request! could you include the alias in the config and use it as an absolute import? Thanks!